<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OVEN_Heater</class>
 <widget class="QMainWindow" name="OVEN_Heater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OVEN_Heater</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_serialparams">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>SERIAL 115200</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_modbusparams">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>MODBUS 127 65535 65535 65535 65535</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_activeport">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>Поиск Устройств</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSerial_Setup">
    <property name="title">
     <string>Конфигурация</string>
    </property>
    <addaction name="action_serial_settings"/>
    <addaction name="action_modbus_settings"/>
   </widget>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_save_experiment"/>
    <addaction name="action_save_experiment_as"/>
    <addaction name="action_load_experiment"/>
    <addaction name="separator"/>
    <addaction name="action_fast_heating"/>
    <addaction name="action_profile_heating"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSerial_Setup"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_load_experiment">
   <property name="text">
    <string>Загрузить Эксперимент</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_fast_heating">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Быстрый Нагрев</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="action_profile_heating">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Профильный Нагрев</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="action_serial_settings">
   <property name="text">
    <string>Настройки Последовательного Порта</string>
   </property>
  </action>
  <action name="action_modbus_settings">
   <property name="text">
    <string>Настройки MODBUS</string>
   </property>
  </action>
  <action name="action_save_experiment">
   <property name="text">
    <string>Сохранить Эксперимент</string>
   </property>
  </action>
  <action name="action_save_experiment_as">
   <property name="text">
    <string>Сохранить Эксперимент как ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
